models/path.py

```diff
@@ -44,14 +44,14 @@ def make_vlans_available(self, controller):
         """Make the VLANs used in a path available when undeployed."""
         for link in self:
             tag = link.get_metadata("s_vlan")
-            result_a, result_b = link.make_tags_available(
-                controller, tag.value, link.id, tag.tag_type
+            conflict_a, conflict_b = link.make_tags_available(
+                controller, tag.value, link.id, tag.tag_type, False
```

Comment:
In places when you need to pass a boolean in the args if you could pass `check_order=False` explicitly that'd improve readability, without having to check the implementation source code to try to remember

Reply:
Added it